pkg/kubelet/kubelet.go

```diff
@@ -1900,7 +1919,11 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	}
 
 	// Wait for volumes to attach/mount
-	if err := kl.volumeManager.WaitForAttachAndMount(pod); err != nil {
+	defaultedPod, _, err := kl.defaultPodLimitsForDownwardApi(pod, nil)
```

Comment:
We really should refactor some aspects of standalone mode handling in Kubelet.  It's kind of a hidden scenario in code base.